static/playGame.html

```diff
@@ -15,22 +15,22 @@
 <div class="container" data-ng-show="!isOpenGame()">
   <div class="eight columns">
     <p data-ng-show="gameIsActive()">
-      You are in an ongoing chess game, the pot is at {{game.pot}}.
+      You are in an ongoing chess game, the pot is at {{game.pot}} Ether.
     </p>
     <p data-ng-show="gameIsWon()">
       You have won this game.
-      <span data-ng-show="getGame().pot > 0">The pot of {{getGame().pot}} is being transferred to you.</span>
-      <span data-ng-hide="getGame().pot > 0">The pot was transferred to you.</span>
+      <span data-ng-show="getGame().pot > 0">The pot of {{getGame().pot}} Ether is being transferred to you.</span>
+      <span data-ng-hide="getGame().pot > 0">The pot of {{getGame().pot}} Ether was transferred to you.</span>
```

Comment:
You would have to store it in the contract. Or you need to get it from the old blocks/transactions.